### PR TITLE
docs(angular): add faq with shallow rendering example

### PR DESCRIPTION
**Pull request by marcioggs**
This adds changes suggested by @timdeschryver at https://github.com/testing-library/angular-testing-library/issues/288#issuecomment-1076475686

### CODE REVIEW

File: docs/angular-testing-library/faq.mdx [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
+id: faq
+title: FAQ
+---
+
+See also the [main FAQ](dom-testing-library/faq.mdx) for questions not specific
+to Angular testing.
+
+<details>
```
Comment (marcioggs) [inline docs/angular-testing-library/faq.mdx:9]: This and the 3rd item are identical copies from React's FAQ.
I think these items should apply to all frameworks.
Should keep them on the Angular FAQ as well?
If positive, should we deduplicate them somehow? Maybe by moving them to the main FAQ or by creating a reusable doc snippet.

**Review by timdeschryver — APPROVED** [review]
Thanks @marcioggs

File: .all-contributorsrc [hunk 2]
```diff
@@ -1835,6 +1835,15 @@
...
       ]
+    },
+    {
+      "login": "marcioggs",
+      "name": "Márcio Gabriel",
+      "avatar_url": "https://avatars.githubusercontent.com/u/2506127?v=4",
+      "profile": "https://github.com/marcioggs",
+      "contributions": [
+        "doc"
+      ]
```
Comment (timdeschryver) [inline .all-contributorsrc:1846]: It just caught my attention that we're missing a contributors entry in the readme.
We'll fix that by using the bot.

```suggestion
```

**Comment by timdeschryver** [comment]
@all-contributors please add @marcioggs for docs

**Pull request merged**